src/features/island/plots/Plot.tsx

```diff
@@ -93,8 +93,7 @@ const isSeasonedPlayer = (state: MachineState) =>
   getBumpkinLevel(state.context.state.bumpkin?.experience ?? 0) >= 60 &&
   // - verified (personhood verification)
   state.context.verified &&
-  // - has active seasonal banner
-  hasVipAccess(state.context.state.inventory);
+  hasReputation({ game: state.context.state, reputation: Reputation.Grower });
```

Comment:
(Recommend) Add back a comment for the reputation as the three comments are helpful when pasting into discord when people ask the criteria (without them having to read the code).